### ISSUE TITLE
Theme manager adds space for entry before created

### ISSUE THREAD

**Issue by D3SOX** [florisboard/florisboard#649]
#### Short description
The theme manager has an empty space when the add attribute modal is open.

#### Steps to reproduce
1. Edit a theme
2. Add an attribute
3. Watch the background

![Screenshot_20210413-174229_FlorisBoard_Beta](https://user-images.githubusercontent.com/24937357/114581339-e879ad00-9c7f-11eb-8317-e073a1db885d.png)

#### Environment information
- FlorisBoard Version: 0.3.11-beta01
- Install Source: GitHub
- Device: OnePlus 8T
- Android version, ROM: 10, crDroid 7.4

**Comment by patrickgold** [comment]
Yes, this is definitely the case, because an empty attribute is added as soon as you press the add button. Don't really see this as a bug though, because the empty attribute is removed if you press cancel and it does in my opinion not change the UX nor the UI that much, because it is in the background.

**Comment by D3SOX** [comment]
Yes this is definitely a minor enhancement. Just something I noticed and wanted to mention.
